testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -998,4 +1026,43 @@ private static void checkProperties(Map<?, ?> properties)
             }
         }
     }
+
+    private static final class AllowSpecificGroupSystemAccessControl
```

Comment:
Hm... sorry. Once you wrote code I realized that you most likely use `TestingAccessControlManager`, see:
 - `TestingAccessControlManager#rowFilter`
 - `TestingAccessControlManager#columnMask`
 - `TestingAccessControlManager#denyIdentityTable`